### ISSUE TITLE
[BUG]  MMOitems integration linking not work

### ISSUE THREAD

**Issue by glutaminus** [PluginBugs/Issues-ItemsAdder#1198]
**Describe the bug**
I'm sorry because I'm not sure if this should count as bug or not.
the thing is I can't link integration of itemsadder and mmoitems as this one. https://www.spigotmc.org/resources/items-mmoitem-example-integration.88351/ 

**To Reproduce**
1. use the addon
2. do as it say on wiki
3. get item
4. mmoitems integration are not work

**Expected behavior**
1. use the addon
2. do as it say on wiki
3. get item
4. mmoitems integration are working with itemsadder attributes.

**Info (please complete the following information):**
 - Server version: (example: purpur 1.16.5)
 - Client version: (example: fabric 1.16.5)

**Special Info**
Please do not write "latest"! Tell us the specific plugin version you are using.
 - ItemsAdder version: (example: 2.4.15)
 - LightAPI Fork version: (example: 3.5.2)
 - LoneLibs version: (example: 1.0.14)
 - ProtocolLib version: (example: 4.7.0)

**Additional context**
basically what i'm trying to do is i want to link ItemsAdder Attribute/events with MMOitems

**Comment by LoneDev6** [comment]
MMOItems version?
They may have changed something in the new updates.
The provided method worked fine at the time of when I wrote the tutorial.

Thanks

**Comment by glutaminus** [comment]
I was using MMOitems 6.6.0 at the time.

**Comment by LoneDev6** [comment]
I tested on 6.6.1 and works fine.
Follow the tutorial exactly and use the command `/iaget mmoitems_example:test` at the end, to get the example item.

![immagine](https://user-images.githubusercontent.com/27242001/134202699-566fe46d-5c9f-48ae-9523-0c7e89dd7f1b.png)
![immagine](https://user-images.githubusercontent.com/27242001/134202718-553de2f6-8c44-4383-a88e-7bb11c2706b6.png)
